guidelines/monai-application-package.md

```diff
@@ -1,498 +1,588 @@
+      - Example: `1.5Gi`, `2048Mi`
 
-| Name                     | Required | Default | Type    | Format                    | Description                                                                  |
-| ------------------------ | -------- | ------- | ------- | ------------------------- | ---------------------------------------------------------------------------- |
-| `api-version`            | No       | 0.0.0   | string  | semantic version          | Version of the manifest file schema.                                         |
-| `application-root`       | **Yes**  | N/A     | string  | absolute file-system path | Absolute file-system path to the folder older which contains the Application |
-| `models`                 | No       | N/A     | array   | array of objects          | Array of objects which describe models in the package.                       |
-| `models[*].name`         | No       | N/A     | string  | map reference name        | Name of the map which conforms to the map naming rules.                      |
-| `models[*].path`         | **Yes**  | N/A     | string  | absolute file-system path | Absolute file-system path to the folder which contains the model.            |
-| `resources`              | No       | N/A     | object  | object                    | Object describing resource requirements for the Application.                 |
-| `resources.cpu`          | No       | 1       | integer | number                    | Number of CPU cores required by the Application.                             |
-| `resources.gpu`          | No       | 0       | integer | number                    | Number of GPU devices required by the Application.                           |
-| `resource.memory`        | No       | 1Gi     | string  | memory size               | The maximum memory required by the Application.                              |
-| `resource.shared-memory` | No       | 64Mi    | string  | memory size               | The maximum shared memory required by the Application.                       |
-| `sdk-version`            | No       | 0.0.0   | string  | semantic version          | Version of the SDK used to generate the manifest.                            |
-| `version`                | No       | 0.0.0   | string  | semantic version          | Version of the package.                                                      |
+  - Integer values MUST be positive and not contain any position separators.
 
+    - Example legal values: `1`, `42`, `2048`
 
-## Executor
+    - Example illegal values: `-1`, `1.5`, `2,048`
 
-The MAP Executor (Executor) provides a shim between the runner of a MAP and the MAP's application. Due to the Executor's shim nature, it can be extended beyond its original intent to provide additional functionality.
+  - Decimal values MUST be positive, rounded to the nearest tenth, use the dot (`.`) character to separate whole and fractional values, and not contain any positional separators.
```

Comment:
Are we sure that there is no need for decimal values with more than one decimal digit - e.g. a quarter = '0.25'